lib/usage-tracking/class-usage-tracking-base.php

```diff
@@ -430,6 +430,14 @@ protected function hide_tracking_opt_in() {
 	 * @return bool true if the opt-in is hidden, false otherwise.
 	 **/
 	protected function is_opt_in_hidden() {
+		$delayed_notice_timestamp  = (int) get_option( 'job_manager_display_usage_tracking_once' );
```

Comment:
Can we maybe move this option name to a constant somewhere?

I see it being used at least 4 times just on this PR, and I think using constants can really help to avoid mistakes/typos/etc.